### PR TITLE
Various tiny fixes

### PR DESCRIPTION
**Pull request by vTurbine**

### CODE REVIEW

File: src/Layers/xrRender/r__sector.h [hunk 1]
```diff
@@ -22,8 +22,17 @@ class CPortal : public IRender_Portal
                 public pureRender
 #endif
 {
+public:
+    using Poly = svector<Fvector, 6>;
```
Comment (drug007) [inline src/Layers/xrRender/r__sector.h:26]: In original version 8 elements, not 6, is it intentional change?

Comment (vTurbine) [reply src/Layers/xrRender/r__sector.h:26]: Serialized portals are stored in a way that they can't have more than 6 vertices (see `b_portal` and loading code around). When loaded, their polygon shape doesn't change anymore. Except for clipping code (see `CSector::traverse()` "Clip by frustum" section) where we can have an additional vertex from intersection with a frustum plane.

Comment (vTurbine) [reply src/Layers/xrRender/r__sector.h:26]: Oh, shi.. `ClipPoly()` re-uses existing container, so probably it's a good idea to increase the size of clipped portal poly.

Comment (drug007) [reply src/Layers/xrRender/r__sector.h:26]: I see

**Review by Xottab-DUTY — APPROVED** [review]

**Pull request merged**